Joining together CTRP data.Rmd

```diff
@@ -0,0 +1,304 @@
+
+experiment_info_rename <- experiment_info %>%
+    rename("Exp._ID" = expt_id, "Date" = experiment_date, "Cells_per_well" = cells_per_well, "Cell_line_ID" = master_ccl_id) %>%
+    mutate(Date = ymd(Date)) %>%
+    select("Exp._ID", "Date", "Cell_line_ID", "Cells_per_well")
+
+table3 <- head(experiment_info_rename)
+kable(table3, align = "cccc")
+
+```
+
+### 3d. Cancer cell line info
+```{r Cancer cell line info}
+
+# Rename the columns in cancer_cell_line_info file
+
+cancer_cell_line_rename <- cancer_cell_line %>%
+    rename("Cell_line_name" = ccl_name, "Cancer_type" = cancer_type, "Cell_line_ID" = master_ccl_id)
+
+table4 <- head(cancer_cell_line_rename)
+kable(table4, align = "cll")
+
+```
+
+### 3e. Compound info
+```{r Compound info}
+
+# Rename the columns in compound_info file
+
+compound_rename <- compound %>%
+    rename("Compound_ID" = master_cpd_id, "Compound_name" = cpd_name, "Target_gene" = gene_symbol_of_protein_target)
+
+table5 <- head(compound_rename)
+kable(table5, align = "clc")
+
+```
+### 3f. Combine the data frames
+```{r Combining data frames}
+
+# Combine the AUC_1 and AUC_2 data frames into one - AUC_info
+
+AUC_info <- rbind(AUC_1_rename, AUC_2_rename)
+
+# Combine experiment_info with cancer_cell_line_info 
+
+experiment_cancer_cell_line <- experiment_info_rename %>%
+      full_join(cancer_cell_line_rename, by = "Cell_line_ID")
+
+# Combine AUC_info and experiment_cancer_cell_line
+
+AUC_exp_cancer <- AUC_info %>%
+      full_join(experiment_cancer_cell_line, by = "Exp._ID")
+
+# Add compound_info to experiment_cancer_cell_line
+
+AUC_exp_cancer_compound <- AUC_exp_cancer  %>%
+     full_join(compound_rename, by = "Compound_ID")
+
+# Rename the column in the combined data frame
+
+all_data <- AUC_exp_cancer_compound %>%
+    rename("Cells/well" = "Cells_per_well")
+
+# Rorder the columns in the combined data frame
+
+all_data<- select(all_data, "Date", "Exp._ID", "Cell_line_name", "Cell_line_ID", "Cancer_type", "Cells/well", "Compound_name", "Compound_ID", "AUC", "Target_gene")
+
+# Replace the NAs in Target gene column
+
+target_gene <- all_data$`Target_gene` %>%
+    replace_na("-")
+
+final_data <- all_data %>%
+    select(-'Target_gene') %>%
+    mutate('Target_gene' = target_gene)
+
+# I'm showing the final table in two parts for better visibility
+
+table6 <- head(final_data[,1:6])
+kable(table6, align = "lccclc")
+    
+table7 <- head(final_data[,7:10])
+kable(table7, align = "lccc")
+
+```
+
+## 4. Questions
+
+### 4a. Which cancer type has the lowest AUC values to the compound "vorinostat"?
+
+```{r Vorinostat}
+
+# Calculate mean AUC for vorinostat used alone or in combination with other drug
+
+vorinostat_mean <- final_data %>%
+    select("Cell_line_name", "Cell_line_ID", "Cancer_type", "Compound_name", "Compound_ID", "AUC") %>%
+    separate(col = "Compound_name", into = c("Compound_1", "Compound_2")) %>%
+    filter(Compound_1 == "vorinostat" | Compound_2 == "vorinostat") %>%
```

Comment:
This is a great way to include the data where vorinostat was used in combination with another drug. I only filtered for vorinostat by itself and didn't consider the difference.